extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+
+$track_title    = $attachment['title'];
+$track_duration = ! empty( $attachment['duration'] ) ? $attachment['duration'] : '';
+
+$class = 'jetpack-podcast-player__track ' . $secondary_colors['class'];
+$style = $secondary_colors['style'];
+if ( $is_active ) {
+	$class = 'jetpack-podcast-player__track is-active ' . $primary_colors['class'];
+	$style = $primary_colors['style'];
+}
+
+?>
+
+<li
+	class="<?php echo esc_attr( $class ); ?>"
+	style="<?php echo esc_attr( $style ); ?>"
+>
+	<a
+		class="jetpack-podcast-player__track-link"
+		href="<?php echo esc_url( $attachment['link'] ); ?>"
+		role="button"
+		<?php echo $is_active ? 'aria-current="track"' : ''; ?>
+	>
+		<span class="jetpack-podcast-player__track-status-icon"></span>
+		<span class="jetpack-podcast-player__track-title"><?php echo esc_html( $track_title ); ?></span>
+		<time class="jetpack-podcast-player__track-duration"><?php echo esc_attr( $track_duration ); ?></time>
+	</a>
+</li>
+
+<?php
+// phpcs:enable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
`phpcs:disable` is only valid for the current file, so it doesn't need to be "re-enabled" at the end of the file